dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+
+{: .table .table-bordered .table-striped }
+| Name     | Scope                                                        | Type   | Description                                                                                    |
+|----------+--------------------------------------------------------------+--------+------------------------------------------------------------------------------------------------|
+| `banner` | Required, unless either of the other properties are present. | Object | Defines properties of a banner ad.  For examples, see [`adUnit.mediaTypes.banner`](#banner).   |
+| `video`  | Required, unless either of the other properties are present. | Object | Defines properties of a video ad.  For examples, see [`adUnit.mediaTypes.video`](#video).      |
+| `native` | Required, unless either of the other properties are present. | Object | Defines properties of a native ad.  For properties, see [`adUnit.mediaTypes.native`](#native). |
+
+<a name="banner" />
+
+#### `adUnit.mediaTypes.banner`
+
+{: .table .table-bordered .table-striped }
+| Name    | Scope    | Type                                  | Description                                                                             |
+|---------+----------+---------------------------------------+-----------------------------------------------------------------------------------------|
+| `sizes` | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`. |
+| `name`  | Optional | String                                | Name for this banner ad unit.  Can be used for testing and debugging.                   |
+
+<a name="native" />
+
+#### `adUnit.mediaTypes.native`
+
+The `native` object contains the following properties that correspond to the assets of the native ad.
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Type                                  | Description                                                                                                                       |
+|---------------+----------+---------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------|
+| `sizes`       | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                           |
+| `type`        | Optional | String                                | Used as a shorthand, e.g., `type: 'image'` implies required fields `image`, `title`, `sponsoredBy`, `clickUrl`.                   |
+| `title`       | Required | Object                                | The title of the ad, usually a call to action or a brand name.  For properties, see [`native.title`](#native-title).              |
```

Comment:
None of the native assets (title, body, sponsoredBy, icon, image, clickUrl, cta) should be required in the mediaTypes.native object.

Reply:
Updated to mark all asset fields as optional in bccf932